public/static/locales/isl/demographics.json

```diff
@@ -2,11 +2,19 @@
     "age": "Aldur",
     "gender": "Kyn",
     "native-tongue": "Móðurmál",
+    "icelandic-proficiency": "Íslenskukunnátta",
     "why-this-matters": "Hvers vegna skiptir þetta máli?",
     "why-this-matters-text": "Ofantaldar upplýsingar eru notaðar til að meta hversu lýðfræðilega dreift gagnasafnið Samrómur er. Því dreifðara og fjölbreyttara sem það er, því betra. Sjá skilmála og persónuverndaryfirlýsingu verkefnisins hér fyrir neðan til þess að fá frekari upplýsingar.",
     "accept-terms-conditions": "Ég staðfesti að hafa kynnt mér <2>skilmála</2> og <6>persónuverndaryfirlýsingu</5> verkefnisins",
     "consent-confirmed": "Leyfi staðfest",
     "switch-user": "Skipta um notanda",
+    "icelandic-proficiencies": {
+        "native": "Hefur móðurmálsþekkingu",
```

Comment:
Should be "Hef móðurmálsþekkingu" for the user.
As in "Ég _hef móðurmálsþekkingu_", similar to "Ég er _byrjandi (A1-A2)_"

Reply:
Changed to "Hef móðurmálsþekkingu" in d9e3f7e22a89f7c602897e1c9a08d73d26765528